TU-Agent/tygron.mod2g

```diff
@@ -6,26 +6,34 @@ use buyLand as module.
 use upgrade as module.
 use handleRequest as module.
 use buildGreen as module.
+use sellLand as module.
+use sellLandHandler as module.
 
 exit=nogoals.
 order=linear.
 
 module tygron {
-	
-	%TODO: parken en groen
-
 	%TODO budget?
 	
-	%TODO check for requestgoals and handle these first so other stakeholders can move on without waiting on our bot, prevents a stalemate%
+	%Every 5 cycles we want to try to sell to another stakeholder.
```

Comment:
Why every 5 cycles? Is that because we want to give a stakeholder time to react to our request?